packages/govuk-frontend-review/src/views/layouts/layout.njk

```diff
@@ -14,7 +14,14 @@
     <script src="/vendor/iframe-resizer/iframeResizer.min.js" defer></script>
     <script>
       document.addEventListener('DOMContentLoaded', function () {
-        window.iFrameResize({}, '.js-component-preview')
+        var $examples = document.querySelectorAll('.js-component-preview')
```

Comment:
Since this is an internal development app, are we able to start using modern syntax (`const`, etc)? I'm not sure how the review app fits into our plans! Non-blocking, since I guess the default assumption is we don't change things till we do so elsewhere.

Reply:
I was thinking it'd be nice to see iframes resize in IE 11 and other older browsers

But I think you've convinced me, plus we always have "open in a new window" as backup